baseline_testing/README.md

```diff
@@ -0,0 +1,40 @@
+# Baseline Testing
+
+This folder contains examples and results for performing testing of baseline controllers for different testcases.
+
+## Baseline Controller and Test Case Implementation
+
+- The ``baseline.py`` module file is utilized for executing the baseline control for each test case. It simplifies the process by setting ``u={}``.
```

Comment:
Add full path to baseline.py, since it is in a different directory.

Reply:
Added